security-analysis/security-analysis-api/src/main/java/com/powsybl/security/tools/SecurityAnalysisTool.java

```diff
@@ -317,7 +318,8 @@ void run(CommandLine line, ToolRunningContext context,
 
         SecurityAnalysisResult result = report.getResult();
 
-        if (!result.getPreContingencyLimitViolationsResult().isComputationOk()) {
+        if (result.getPreContingencyResult() == null ||
```

Comment:
In SecurityAnalysisToolTest.testRunWithLog result is not null but all the members are null (because it is actually a mock). I think the real fix lies somewhere else.